(sakaki: fix a few instances of char* being passed to a __u8* without cast)
---
--- a/EepromDev.cpp
+++ b/EepromDev.cpp
@@ -185,12 +185,12 @@ bool EepromDev::readI2C(int fd, int addr, void *data, int count)
     messages[0].addr = EepromAddress;
     messages[0].flags = 0;
     messages[0].len = sizeof(set_addr_buf);
-    messages[0].buf = set_addr_buf;
+    messages[0].buf = (unsigned char *) set_addr_buf;
 
     messages[1].addr = EepromAddress;
     messages[1].flags = I2C_M_RD;
     messages[1].len = count;
-    messages[1].buf = (char *) data;
+    messages[1].buf = (unsigned char *) data;
 
     session.msgs = messages;
     session.nmsgs = 2;
@@ -218,7 +218,7 @@ bool EepromDev::writeI2C(int fd, int addr, void *data, int count)
     messages[0].addr = EepromAddress;
     messages[0].flags = 0;
     messages[0].len = sizeof(data_buf);
-    messages[0].buf = data_buf;
+    messages[0].buf = (unsigned char *) data_buf;
 
     session.msgs = messages;
     session.nmsgs = 1;
